frontend/app/become-artisan/page.tsx

```
@@ -0,0 +1,141 @@
+            toast.error("Failed to become an artisan. Please try again.");
+            console.error(error);
+        }
+    });
+
+    return (
+        <div className="container mx-auto py-10">
+            {
+                mutation.isPending && (
+                    <div className="container mx-auto py-10">
+                        <div className="flex items-center justify-center">
+                            <div className="w-10 h-10 border-t-transparent border-b-transparent border-r-transparent border-l-blue-500 rounded-full animate-spin"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)